listings/applications/erc721/src/erc721.cairo

```diff
@@ -19,9 +19,9 @@ pub trait IERC721<TContractState> {
 
 #[starknet::interface]
 pub trait IERC721Metadata<TContractState> {
```

Comment:
Please do not resolve comments that have not been addressed.
This trait should be implemented, as per the standard.